node/res/bp_parachain.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Zeitgeist Battery Park",
+  "id": "battery_park",
+  "chainType": "Live",
+  "bootNodes": [
+    "/ip4/139.162.171.58/tcp/30333/p2p/12D3KooWPvu5rpH2FNYnAmiQ8X8XqkMiuSFTjH2jwMCSjoam7RGQ"
+  ],
+  "telemetryEndpoints": [
+    [
+      "/dns/telemetry.polkadot.io/tcp/443/x-parity-wss/%2Fsubmit%2F",
```

Comment:
Should this not be:
> wss://telemetry.polkadot.io/submit/

? In case of yes, it also has to be changed in node/res/bp.json

Reply:
Should be fine. It's a valid multiaddress and was also what was generated from the chainspec generation from the rust code. Will test it out though